.github/ISSUE_TEMPLATE/config.yml

```
@@ -7,5 +7,5 @@ contact_links:
     url: https://developercommunity.visualstudio.com
     about: Feature requests are tracked on Visual Studio Developer Community because it provides good prioritization features like voting. Filing there helps ensure new features are properly considered.
   - name: "🙋‍♀️ Community Support"
-    url: https://docs.microsoft.com/answers/topics/164819/dotnet-xamarin.html
+    url: https://learn.microsoft.com/en-us/answers/tags/18/xamarin
```

I think we should remove the `en-us`, it's a thing the VS Code doc extension warns about:
```suggestion
    url: https://learn.microsoft.com/answers/tags/18/xamarin
```